NovemberQA/NovemberQA/Feature/Language.feature

```diff
@@ -19,12 +19,16 @@ Examples:
 | Hindi     | Conversational    |      
 
 @editlanguages
-Scenario: Edit Languages in the Profile
-   When I edit Languages and Level to the profile
-   Then The Languages and Level should be edited successfully
+Scenario Outline: A2 Edit Languages in the Profile
+   When I edit '<Languages>' and '<Level>' to the profile
+   Then The '<Languages>' and '<Level>' should be edited successfully
+
+Examples: 
+| Languages | Level          |
+| French    | Conversational |
 
 @deletelanguages
-Scenario: Delete Languages in the Profile
+Scenario: A3 Delete Languages in the Profile
      When I delete Languages and Level from the profile
      Then Languages and Level should be deleted successfully
 
```

Comment:
furture reference can add table here too, maybe for the tasks ahead you can add this